packages/contracts/script/Verify.ts

```diff
@@ -0,0 +1,44 @@
+import { exec } from "child_process";
+
+const verifyArbitrationToken = async (address: string, path: string) => {
+  exec(
+    ["forge verify-contract", address, path, "--constructor-args", "000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000012000000000000000000000000000000000000000000000000000000000000000b436f75727420546f6b656e00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024354000000000000000000000000000000000000000000000000000000000000", "--chain-id", 11155111, "--etherscan-api-key", "6EI6Z7T8UAAVRIY8JXRXKVKM5A1WH8PURD"].join(" "),
```

Comment:
Where are these `--constructor-args` coming from?

Reply:
We can get it on etherscan. But yeah, I sent a wrong token argument, so I need to deploy again and update README file as well